python/lsst/daf/butler/cli/cliLog.py

```diff
@@ -86,6 +87,29 @@ class CliLog:
     """Any FileHandler classes attached to the root logger by this class
     that need to be closed on reset."""
 
+    @staticmethod
+    def root_loggers() -> Set[str]:
+        """Return the default root logger.
+
+        Returns
+        -------
+        log_name : `set` of `str`
+            The name(s) of the root logger(s) to use when the log level is
+            being set without a log name being specified.
+
+        Notes
+        -----
+        The default is ``lsst`` (which controls the butler infrastructure)
+        but additional loggers can be specified by setting the environment
+        variable ``DAF_BUTLER_ROOT_LOGGER``. This variable can contain
+        multiple default loggers separated by a ``:``.
+        """
+        log_names = set(["lsst"])
+        envvar = "DAF_BUTLER_ROOT_LOGGER"
+        if envvar in os.environ:
+            log_names |= set(os.environ[envvar].split(":"))
```

Comment:
This should be testable, as should the `__root__=` code, no?

Reply:
Yes. 😦 I will take another look. The code is fairly complicated because it creates a special click command designed solely for reporting log states. I'd have to add a slightly different one.